index.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width">
     <title>Imprint</title>
     <link rel="stylesheet" href="app.css">
-    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" />
+    <link rel="stylesheet" href="bootstrap.min.css">
```

Comment:
Please load CDN-based bootstrap css instead of embedding here. Better page loading times this way:

https://www.bootstrapcdn.com/